### PR TITLE
[RA2] ch01 updates to address scope, include k8s cluster management

### LINKED ISSUE #449: [RA2 Core]: build a consensus on Kubernetes cluster management and document the result
In the discussions of #383 there was a disagreement on two topics in the area of Kubernetes cluster management: 

- Is CaaS Manager part of NFVO and if we care about the result in RA2
- Scope of one Kubernetes cluster. One VNF, group of VNF from the same vendor, group of VNF-s

The resolution of this issue shall clarify these and document the result.

### PR DESCRIPTION
**Pull request by tomkivlin**
Added content to chapter 1, including scope information.  Have updated the status in README.md and have created an appendix to be used for general VNF-to-CNF transition guidance.

Fixes #449

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter01.md [hunk 1]
```diff
@@ -73,56 +82,40 @@ The definition above is very application centric and look at cloud native from t
...
-
-<p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
-<p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
-
-### A note on virtualised and containerised workloads
-It is possible that a single VNFM will be managing applications (or components) that are virtualised (i.e. running in VMs) and applications (or components) that are containerised (i.e. running in containers), at the same time.  There are different approaches to achieving this:
-1. Application manager (EMS, VNFM) uses both IaaS (VIM) API and Kubernetes API - the former for VM based workloads, the latter for containerised workloads.
-2. Application manager (EMS, VNFM) uses just the Kubernetes API for both VM-based and container-based workloads, with the Kubernetes PaaS managing the lifecycle of VMs using on of the following methods:
-    a) Kubernetes interacts with IaaS/VIM API (Cluster API model)
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter01.md]: Are there any solutions for this?

**Comment by rabiabdel** [comment]
what is the current status of this @tomkivlin

File: doc/ref_arch/kubernetes/chapters/chapter01.md (hunk 1 above)
Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter01.md]: @CsatariGergely for the removed content do you mean?  Given this is removed for now, do we need to change anything?

**Comment by tomkivlin** [comment]
> what is the current status of this @tomkivlin

Awaiting reviews / approvals from the pending reviewers (apart from Gergely, who has commented).

**Comment by rabi-abdel** [comment]
@TamasZsiros @yangxu-hw can you please review and approved if you are happy.

File: doc/ref_arch/kubernetes/chapters/chapter01.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter01.md]: Oh, yes sorry for the removed content. No need to change.

**Comment by tomkivlin** [comment]
@TamasZsiros thanks - have made that change (but wrongly referenced @CsatariGergely in the commit. It's been a long day...)

**Comment by rabi-abdel** [comment]
@CsatariGergely @pgoyal01 can you please review?

**Pull request merged**